inst/ms/aggreCAT.qmd

```diff
@@ -169,16 +201,17 @@ datasets collected using such elicitation methods.
 
 We begin by formulating the problem of mathematically aggregating expert
 judgements. Each method, and its data requirements is summarised in
-table X (cross-ref). We also briefly summarise package datasets, which
-were collected by the repliCATS project. By first describing the
-datasets before describing the aggregation methods in detail, we aim to
-provide a grounded understanding of the different outputs of expert
-elicitation using the repliCATS IDEA protocol, and the inputs available
-to the aggregation functions.
+Table \ref{tbl-method-summary-table}. Before outlining key aggregation methods, we
```

Comment:
L179: I wouldn't italicise Confidence Score here since you haven't earlier, and it confuses t with the IDEA steps here.
L184: X = 900 single trace + 100 bushel in phase two